scripts/source-maintenance.sh

```diff
@@ -61,10 +71,22 @@ else
 	MD5="md5sum"
 fi
 
-ASVER=`astyle --version 2>&1 | grep -o -E "[0-9.]+"`
-if test "${ASVER}" != "2.04" ; then
-	echo "Astyle version problem. You have ${ASVER} instead of 2.04"
-	ASVER=""
+${ASTYLE} --version >/dev/null 2>/dev/null
+result=$?
+if test $result -gt 0 ; then
+	echo "ERROR: cannot run ${ASTYLE}"
+	exit 1
+fi
+ASVER=`${ASTYLE} --version 2>&1 | grep -o -E "[0-9.]+"`
+if test "${ASVER}" != "${TargetAstyleVersion}" ; then
+	if test "${ASTYLE}" = "astyle" ; then
+		echo "Astyle version problem. You have ${ASVER} instead of ${TargetAstyleVersion}"
+		echo "Formatting step skipped due to version mismatch"
+		ASVER=""
+	else
+		echo "WARNING: ${ASTYLE} is version ${ASVER} instead of target ${TargetAstyleVersion}"
```

Comment:
See my older comments regarding "target" and other mysterious (to uninitiated script users) descriptions of TargetAstyleVersion.

Reply:
added comments and removed the word "target" from the message